src/docs/about.md

```diff
@@ -0,0 +1,186 @@
+---
+title: What is Broccoli?
+---
+
+Broccoli is a simple Javascript build tool, that allows you to configure your build pipeline in Javascript (like you're
+already used to writing), and handles setting up the filesystem state for each transformation that's going to happen.
+
+${toc}
+
+## What is a build tool?
+
+A build tool's job is to take input files (your javascript, css, html, etc), to process them and output the result of
+those transformations, usually into some distributable form. Typically this will involve things like javascript 
+transformations to allow you to write newer syntax that will work in a browser, to use things like Sass for your CSS,
+etc.
+
+Broccoli.js is different to other build tools. You may be used to tools like [Grunt](https://gruntjs.com/) (a task
+runner), [Gulp](https://gulpjs.com/) (streams and pipes) or [Webpack](https://webpack.js.org/) (a module bundler),
+these all operate on different levels to Broccoli.
+
+Broccoli works at the file-system level, it provides a Javascript API to wrap other node tools like
+[Babel](https://babeljs.io/), [Rollup](https://rollupjs.org/) or [Node-Sass](https://github.com/sass/node-sass), that
+operate on a set of input files, and emit output files, that is all. Broccoli has no knowledge of the contents of 
+your files, only input and output directories.
+
+## Thinking in Broccoli
+
+Broccoli at its heart is very simple. It's really just a set of connected file transformations, performed by plugins,
+that manipulate files from one or multiple inputs, into a single output. Each plugin is responsible for processing files
+passed to it in input directories, and writing files to its output directory.
+
+Broccoli handles managing the file-system state, and simply takes input files and passes them as inputs to plugins, 
+and takes the output from each plugin and passes it as an input to the next one. The order in which these operations 
+happen is determined by the build file, a `Brocfile.js` that you're going to learn how to create.
+
+You can think of Broccoli much like a simple programming language, where the output of a function can be passed as the
+input(s) to another function.
+
+E.g.:
+
+```js
+module.exports = merge([
+    uglify(
+        babel('src')
+    ),
+    autoprefixer(
+        sass('src/styles', 'site.scss')
+    )
+]);
+```
+
+In the above, working from inner to outer, a `src` directory is passed to the `babel()` plugin (which will convert our
```

Comment:
Rather than saying "from inner to outer" might it be clearer to have a number list of steps which sequentializes the pipeline in question? I at-least find "inner to outer" to be hard to follow.